### ISSUE TITLE
avoid the need to specify both source and dest

### ISSUE THREAD

**Issue by bltierney** [perfsonar/pscheduler#175]
Currently when doing assist mode, you have to specify both source and dest, such as:

psc task --assist=antg-dev.es.net throughput --dest slac-pt1.es.net --source antg-dev.es.net

I suggest the local host is assumed when not specified.

eg:

psc task --assist=antg-dev.es.net throughput --dest slac-pt1.es.net  

is the same as:

psc task --assist=antg-dev.es.net throughput --dest slac-pt1.es.net --source local_hostname

and

psc task --assist=antg-dev.es.net throughput --source antg-dev.es.net

is the same as:

psc task --assist=antg-dev.es.net throughput --dest local_hostname --source  antg-dev.es.net

And perhaps can be --assist can assume localhost too?

eg: just type this:
psc task --assist throughput --dest antg-dev.es.net --dest slac-pt1.es.net 

to get:

psc task --assist=antg-dev.es.net throughput --dest antg-dev.es.net --dest slac-pt1.es.net

**Comment by arlake228** [comment]
I don't agree with this and its not "easy" since it requires the dumb client to know something about the plug-in its running i.e have the plug-in installed, which by definition it doesn't since you are using assist. If you want this behavior you have to install the full pscheduler-server on the host, if you don't then you have to specify things like --source to be unambiguous.

**Comment by mfeit-internet2** [comment]
I told Brian to put the `easy` tag on this, because it isn't difficult to implement.

We had a discussion about ambiguity and assistance some time ago, but I think it was on video, so there isn't any record of what was said.  (I think it was the same call where we went over the BWCTL-to-pScheduler hint list.)   Anyway, my recollection is that we decided that if using an assist server, there shouldn't be ambiguity in the test spec.

The hairy point -- and I just thought of this -- is that if you say `psc task --source there` and the participant list diviner assumes that `dest` will be `here`, it will get what the current host thinks its FQDN is and pass that to `there`, which will be the lead.  My development VMs know themselves by hostnames like `dev7`, and there's no FQDN, nor is there a way for the outside to connect to it.  Ergo, if you force specificity, there's no way for that situation to happen.

Based on that, I think we should leave things as they are.
